core/plugins/org.polarsys.capella.core.sirius.analysis/src/org/polarsys/capella/core/sirius/analysis/TitleBlockServices.java

```diff
@@ -715,6 +715,7 @@ public Object getCellContent(EObject diagram, EObject cell, EObject containerVie
         if (feature != null) {
           Object obj = TitleBlockHelper.getResultOfExpression(RepresentationHelper.getRepresentationDescriptor((DDiagram) diagram),
               feature, titleBlockContainer);
+          obj = ((String) obj).replaceAll("\\<[^>]*>","");

```

Comment:
Is not the good place here because obj can be another type and not string and the cast will fail.
A better place to add this is in getWrappedObject, when we call obj.toStrin() (is called twice)